examples/trip.rs

```diff
@@ -1,12 +1,10 @@
-//!
```

Comment:
Ha -- I had learned at one point the style rule that, for multi-line line comments, always include a blank line before and after.  I'm not sure where this came from.  It seems like we don't prefer that so I'll stop doing it.